src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/UnsafeNativeMethods.cs

```diff
@@ -419,275 +363,247 @@ internal enum EvtSeekFlags
+                            [MarshalAs(UnmanagedType.LPWStr)] string query,
                             int flags);
 
         // SEEK
-        [DllImport(WEVTAPI, CharSet = CharSet.Auto, SetLastError = true)]
-        internal static extern bool EvtSeek(
+        [GeneratedDllImport(Interop.Libraries.Wevtapi, SetLastError = true)]
+        internal static partial bool EvtSeek(
                             EventLogHandle resultSet,
                             long position,
                             EventLogHandle bookmark,
                             int timeout,
-                            [MarshalAs(UnmanagedType.I4)]EvtSeekFlags flags
-                                        );
+                            EvtSeekFlags flags);
 
-        [DllImport(WEVTAPI, CallingConvention = CallingConvention.Winapi, SetLastError = true)]
-        internal static extern EventLogHandle EvtSubscribe(
+        [GeneratedDllImport(Interop.Libraries.Wevtapi, SetLastError = true)]
+        internal static partial EventLogHandle EvtSubscribe(
                             EventLogHandle session,
                             SafeWaitHandle signalEvent,
-                            [MarshalAs(UnmanagedType.LPWStr)]string path,
-                            [MarshalAs(UnmanagedType.LPWStr)]string query,
+                            [MarshalAs(UnmanagedType.LPWStr)] string path,
+                            [MarshalAs(UnmanagedType.LPWStr)] string query,
                             EventLogHandle bookmark,
                             IntPtr context,
                             IntPtr callback,
                             int flags);
 
-        [DllImport(WEVTAPI, CallingConvention = CallingConvention.Winapi, SetLastError = true)]
-        internal static extern bool EvtNext(
+        [GeneratedDllImport(Interop.Libraries.Wevtapi, SetLastError = true)]
+        internal static partial bool EvtNext(
                             EventLogHandle queryHandle,
                             int eventSize,
                             [MarshalAs(UnmanagedType.LPArray)] IntPtr[] events,
                             int timeout,
                             int flags,
                             ref int returned);
 
-        [DllImport(WEVTAPI, CallingConvention = CallingConvention.Winapi, SetLastError = true)]
-        internal static extern bool EvtCancel(EventLogHandle handle);
+        [GeneratedDllImport(Interop.Libraries.Wevtapi, SetLastError = true)]
+        internal static partial bool EvtCancel(EventLogHandle handle);
 
-        [DllImport(WEVTAPI)]
-        internal static extern bool EvtClose(IntPtr handle);
+        [GeneratedDllImport(Interop.Libraries.Wevtapi)]
+        internal static partial bool EvtClose(IntPtr handle);
 
         /*
-        [DllImport(WEVTAPI, EntryPoint = "EvtClose", SetLastError = true)]
+        [DllImport(Interop.Libraries.Wevtapi, EntryPoint = "EvtClose", SetLastError = true)]
         public static extern bool EvtClose(
```

Comment:
Isn't this the same as above?

Reply:
This is commented out (hit it with my find/replace). Will remove.